src/sentry/static/sentry/app/components/events/interfaces/frame/utils.tsx

```diff
@@ -62,3 +62,7 @@ export function getFrameHint(frame: Frame) {
 
   return [null, null];
 }
+
+export function isCsharp(platform: string) {
```

Comment:
what about naming this function `isDotnet` ?

platform=csharp actually represents `.NET` and can be `F#`, `VB` or any language targeting CLS (the Common Language Specification).

Maybe this ^ as a comment inside the function

Reply:
sounds good, you got it 👍